alt_palindrome_permutation.js

```diff
@@ -0,0 +1,56 @@
+function permPalindrome(phrase) {
+    //TODO: could use const instead of var because the value doesn't change
+    //-cleaner
+    //-readability
+    //-prevents bugs
+    const countMap = buildCharFrequencyTable(phrase);
+    return checkMaxOneOdd(countMap);
+}
+
+function checkMaxOneOdd(countMap) {
+    var count = 0;
+    for (i of countMap.values()) {
+        // % = modulus operation, similar to division but gives a remainder 
+        //if anything besides 0, it is odd => enter into this if block
+        if (i % 2) {
+            count += 1;
+        }
+        if (count >1) {
+            return false;
+        }
+    }
+    return true; //that it is a permutation
+}
```

Comment:
While there's nothing wrong with your approach here, this is also a good place to practice using JS's `Array.prototype.reduce` function. Because `countMap.values()` will return an iterator, you can spread (`...`) the result into an array to get an array. You can then use the `reduce` function, which condenses an array into a single value, in this case, into the number of odd values. You can then compare this result against `1` like you're already doing.
```suggestion
    return [...countMap.values()].reduce((count, value) => {
      return value % 2 === 1 ? count + 1 : count;
    }, 0) <= 1;
```